src/cogent3/core/sequence.py

```diff
@@ -1909,6 +1909,41 @@ def offset(self) -> int:
     def offset(self, value: int):
         self._offset = value or 0
 
+    @property
+    def absolute_start(self) -> int:
+        """returns the plus strand absolute start
```

Comment:
Needs a more informative doctoring. It's complex for two reasons:
1. because "absolute" has alternate meanings in maths compared to general language. 
2.  because `absolute_start` is wrt to the underlying seq, unless there has been an offset set for the underlying annotations in which case it is wrt to the coordinates of the annotations.

Reply:
Good points. Changing the name is probably best bet. Which of the following seems best
- `parent_start`
- `full_start`
- `resolved_start`
- `adjusted_start`
or something else?

Reply:
I think `parent_start`